samples/bluetooth/iso_broadcast/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_ADV_ISO=y
```

Comment:
Is the
`CONFIG_BT_CTLR_SYNC_PERIODIC` also needed?

See tests/bluetooth/shell/boards/*conf